src/__tests__/dbOperationAuditLogs.int.test.ts

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, test } from '@jest/globals';
+import { db } from '../database/db';
+import { createDbOperationAuditLog } from '../database/operations/dbOperationAuditLogs/createDbOperationAuditLog';
+import { loadUnifiedAuditLogBundle } from '../database/operations/unifiedAuditLogs';
+import {
+	expectTimestamp,
+	getTestAuthContext,
+	NO_LIMIT,
+	NO_OFFSET,
+} from '../test/utils';
+
+describe('db operation audit logs appear', () => {
+	test('direct call to createDbOperationAuditLog appears', async () => {
```

Comment:
We use `it` instead of `test` in all our other tests FWIW -- is there a reason to start using `test` here?

Reply:
Resolved in #1630 .